kernel/data-pipeline/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/util/ColumnValueConvertUtilsTest.java

```diff
@@ -105,4 +107,11 @@ void assertConvertToProtobufMessage() {
         assertThat(((com.google.protobuf.Timestamp) actualMessage).getSeconds(), is(offsetDateTime.toEpochSecond()));
         assertThat(((com.google.protobuf.Timestamp) actualMessage).getNanos(), is(offsetDateTime.getNano()));
     }
+    
+    @Test
+    void assertTimeConvert() {
+        Time time = new Time(-3612 * 1000);
+        Int64Value actualMessage = (Int64Value) ColumnValueConvertUtils.convertToProtobufMessage(time);
+        assertThat(LocalTime.ofNanoOfDay(actualMessage.getValue()), is(time.toLocalTime()));
```

Comment:
If `time.toLocalTime()` will lose nano time, then it could not verify nano, it's better to verify it in unit test

Reply:
Improved